platform/forms/res/templates/form.html

```diff
@@ -19,35 +19,29 @@
  this source code distribution or the Licensing information page available
  at runtime from the About dialog for additional information.
 -->
-<form name="mctForm" novalidate>
-
-<div class="form">
+<form name="mctForm" novalidate class="form">
     <span ng-repeat="section in structure.sections">
         <div class="section-header" ng-if="section.name">
             {{section.name}}
         </div>
-        <div class="form-section">
+        <div class="form-section" ng-class="{ first:$index < 1 }">
```

Comment:
better to use `$first` or better yet `:first-child` (e.g. `.section:first-child > .form-section { /*css*/};`).